pkg/3scale/amp/operator/redis_reconciler.go

```diff
@@ -60,25 +60,39 @@ func (r *RedisReconciler) Reconcile() (reconcile.Result, error) {
 		return reconcile.Result{}, err
 	}
 
-	dcMutator := reconcilers.DeploymentConfigMutator(
-		reconcilers.DeploymentConfigImageChangeTriggerMutator,
-		reconcilers.DeploymentConfigContainerResourcesMutator,
-		reconcilers.DeploymentConfigAffinityMutator,
-		reconcilers.DeploymentConfigTolerationsMutator,
-		reconcilers.DeploymentConfigPodTemplateLabelsMutator,
-		reconcilers.DeploymentConfigPriorityClassMutator,
-		reconcilers.DeploymentConfigTopologySpreadConstraintsMutator,
-		reconcilers.DeploymentConfigPodTemplateAnnotationsMutator,
-		// 3scale 2.13 -> 2.14
-		upgrade.Redis6CommandArgsEnv,
```

Comment:
Q:
What's the impact on 2.13 > 2.15 if this is removed? Not that we support multi-minor upgrades but just curious.

Reply:
I'll be honest I'm not sure what the impact would be I haven't tested it but I can. My understanding was that we don't support multi-minor upgrades like you said and so it could be safely removed after 2.14

Reply:
I would keep this in then if it's not an issue

Reply:
actually, after a call with Carl, we are dropping this.